### ISSUE TITLE
Creating tree and circle visualizations with the Cli?

### ISSUE THREAD

**Issue by jhercher** [sparna-git/skos-play#65]
Hi,

I'm using the SkosPlay on the [Cli (v0.9)](https://github.com/sparna-git/skos-play/releases/tag/0.9), kudos for your great work!! It looks like only the website could build the tree and circle visualizations for now. Is this planned for a future release, or do I just miss an option on the command line? 

Best,
 Johannes

**Comment by tfrancart** [comment]
No, you don't miss an option, these visualizations are just not packaged in the cli.
I don't have a plan to add them.
